app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/index.tsx

```diff
@@ -85,8 +86,22 @@ export const ChangeApprovalOperations: React.FC<
           className="bg-transparent text-success"
         >
           <p className="my-1" data-testid="in-review-label">
-            <Check className="align-top" /> All set! This experiment will{" "}
-            {actionDescription} as soon as it is approved.
+            Please ask someone on your team with review privileges, or a
+            qualified{" "}
+            <LinkExternal href={EXTERNAL_URLS.EXPERIMENTER_REVIEWERS}>
+              reviewer
```

Comment:
Maybe consider making "qualified reviewer" the link instead of just "reviewer"? (either way is fine by me, just gives larger clickable space)